frontend/src/components/Session/LoginFormContainer.jsx

```diff
@@ -8,7 +8,8 @@ const mapStateToProps = (state, ownProps) => {
   return {
     errors: state.errors,
     formType: 'login',
-    loggedIn: state.session.id !== null ? true : false
+    loggedIn: state.session.id !== null ? true : false,
```

Comment:
viết ngắn gọn như này được ko? `!state.session.id`

Reply:
không anh, nó khác nhau hoàn toàn mà. ví dụ id = null, thì !id = true, còn dòng code trên nó sẽ trả về false